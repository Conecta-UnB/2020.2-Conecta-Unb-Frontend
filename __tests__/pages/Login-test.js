import React from 'react';
import { render } from '@testing-library/react';
import LoginPage from '../../src/pages/Login';

describe('testando loginforms', () => {
  test('deve ser igual a snapshot', () => {
    const { asFragment } = render(<LoginPage />);
    expect(asFragment(<LoginPage />)).toMatchSnapshot();
  });
});
