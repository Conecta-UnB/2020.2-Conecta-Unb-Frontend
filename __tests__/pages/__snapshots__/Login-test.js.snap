// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testando loginforms deve ser igual a snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="underHeader"
      >
        <div
          class="containerLoginForm"
        >
          <div
            class="imgContainer"
          >
            <img
              alt="logoImage"
              src="[object Object]"
            />
          </div>
          <form
            action="#"
          >
            <label
              for="email"
            >
              E-mail:
              <input
                id="email"
                name="email"
                type="email"
                value=""
              />
            </label>
            <label
              for="senha"
            >
              Senha:
              <input
                id="senha"
                name="senha"
                type="password"
                value=""
              />
            </label>
            <button
              type="submit"
            >
              Entrar
            </button>
          </form>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="underHeader"
    >
      <div
        class="containerLoginForm"
      >
        <div
          class="imgContainer"
        >
          <img
            alt="logoImage"
            src="[object Object]"
          />
        </div>
        <form
          action="#"
        >
          <label
            for="email"
          >
            E-mail:
            <input
              id="email"
              name="email"
              type="email"
              value=""
            />
          </label>
          <label
            for="senha"
          >
            Senha:
            <input
              id="senha"
              name="senha"
              type="password"
              value=""
            />
          </label>
          <button
            type="submit"
          >
            Entrar
          </button>
        </form>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
